lib/bugsnag/api/client.rb

```diff
@@ -140,6 +146,7 @@ def token_authenticated?
       def agent
         @agent ||= Sawyer::Agent.new(configuration.endpoint, sawyer_options) do |http|
           http.headers[:content_type] = "application/json"
+          http.headers[:'X-Version'] = "2"
```

Comment:
If I remember this was actually necessary for something (I think the auth-tokens require it?) which introduces a conflict with the issue above.

Reply:
I think it was required before v2 was the default API

Reply:
The default api version is still 1 so this is required

Reply:
This causes a 500 though:

```sh
  curl -H 'Authorization: token [token]' \
       -H 'Content-Type: application/json' \
       -H 'X-Version: 2' \
       https://api.bugsnag.com/accounts/5919dd35488ed9001b53f3c3/projects
```

while removing the `X-Version` header does not. I'm not sure what's happening here since it doesn't seem to be affecting the website's API requests.